python/samples/kernel-syntax-examples/azure_chat_gpt_with_data_api.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+import semantic_kernel as sk
+import semantic_kernel.connectors.ai.open_ai as sk_oai
+
+kernel = sk.Kernel()
+
+# Load Azure OpenAI Settings
+deployment, api_key, endpoint = sk.azure_openai_settings_from_dot_env()
+
+# Load Azure OpenAI with data settings
+azure_aisearch_datasource_settings = (
+    sk.azure_aisearch_datasource_settings_from_dot_env_as_dict()
+)
+azure_chat_with_data_settings = sk.PromptTemplateWithDataConfig.AzureChatWithDataSettings(
+    data_source_parameters=sk.PromptTemplateWithDataConfig.AzureAISearchDataSourceParameters(
+        **azure_aisearch_datasource_settings
+    )
+)
+
+# For example, AI Search index may contain the following document:
+
+# Emily and David, two passionate scientists, met during a research expedition to Antarctica.
+# Bonded by their love for the natural world and shared curiosity, they uncovered a
+# groundbreaking phenomenon in glaciology that could potentially reshape our understanding of climate change.
+
+
+chat_service = sk_oai.AzureChatCompletion(
+    base_url=f"{str(endpoint).rstrip('/')}/openai/deployments/{deployment}/extensions",
```

Comment:
maybe we should move this into a constructur with a flag for the extentions (`if use_extentions: base_url = ...`)